plugins/inputs/cpufreq/README.md

```diff
@@ -0,0 +1,37 @@
+# CPUFreq Input Plugin For Telegraf Agent
+
+The CPUFreq plugin collects the current CPU's frequency. This plugin work only with linux.
+
+## Configuration
+
+```toml
+[[inputs.cpufreq]]
+  ## Path for sysfs filesystem.
+  ## See https://www.kernel.org/doc/Documentation/filesystems/sysfs.txt
+  ## Defaults:
+  # path_sysfs = "/sys"
+  ## Gather CPU throttles per socker
+  ## Defaults:
+  # throttles_per_socket = false
+  ## Gather CPU throttles per physical core
+  ## Defaults:
+  # throttles_per_core = false
+```
+
+## Example output
+
+```
+> cpufreq,cpu=0,host=server01 cur_freq=3756293000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=1,host=server01 cur_freq=3735119000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=2,host=server01 cur_freq=3786381000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=3,host=server01 cur_freq=3823190000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=4,host=server01 cur_freq=3780804000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=5,host=server01 cur_freq=3801758000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=6,host=server01 cur_freq=3839194000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq,cpu=7,host=server01 cur_freq=3877989000,max_freq=3900000000,min_freq=800000000 1527789803000000000
+> cpufreq_cpu_throttles,cpu=0,host=server01 count=0 1527789803000000000
+> cpufreq_core_throttles,core=0,cpu=0,host=server01 count=0 1527789803000000000
+> cpufreq_core_throttles,core=1,cpu=0,host=server01 count=0 1527789803000000000
+> cpufreq_core_throttles,core=2,cpu=0,host=server01 count=0 1527789803000000000
+> cpufreq_core_throttles,core=3,cpu=0,host=server01 count=0 1527789803000000000
```

Comment:
I think you're contradicting yourself. Your comment right above says to merge `cpufreq_core_throttles` with `cpufreq`. But this comment says to just rename `cpufreq_core_throttles` to `cpufreq_core` :-)

I'm in favor of merging both into the `cpufreq` measurement. No need for 2 measurements.

Reply:
I was thinking there would be two measurements: `cpufreq` and `cpufreq_core`.  `cpufreq_core` would have a `core` and `cpu` tag while `cpufreq` would only have `cpu`.

If we have this, then it is hard to select just the core throttling because you have to exclude the item without the tag:
```
cpufreq,cpu=0 throttle=42i,cur_freq=3756293000i
cpufreq,cpu=0,core=0 throttle=42i
```

This could maybe work, but not sure if I like the naming as much:
```
cpufreq,cpu=0 throttle=42i,cur_freq=3756293000i
cpufreq,cpu=0,core=0 core_throttle=42i
```

And here is an example of what I was thinking originally:
```
cpufreq,cpu=0 throttle=42i,cur_freq=3756293000i
cpufreq_core,cpu=0,core=0 throttle=42i
```